test/mocha/benchmark.js

```diff
@@ -0,0 +1,18 @@
+var assert = require("assert");
+var exec = require("child_process").exec;
+
+describe("test/benchmark.js", function() {
+    this.timeout(300000);
+    var command = '"' + process.argv[0] + '" test/benchmark.js ';
+    [
+        "-mc warnings=false",
+//        "-mc keep_fargs=false,passes=3,pure_getters,unsafe,unsafe_comps,unsafe_math,unsafe_proto,warnings=false",
```

Comment:
Commented out pending #1565.

@kzc any additional combinations of options you'd like to see here? I guess we can't go full blast since it would make `npm test` too slow?